internal/kafka/internal/services/data_plane_cluster.go

```diff
@@ -52,7 +52,7 @@ func (d *dataPlaneClusterService) GetDataPlaneClusterConfig(ctx context.Context,
 
 	return &dbapi.DataPlaneClusterConfig{
 		Observability: dbapi.DataPlaneClusterConfigObservability{
-			AccessToken: d.ObservabilityConfig.ObservabilityConfigAccessToken,
+			AccessToken: d.ObservabilityConfig.DataPlaneObservabilityConfig.GithubResourcesAuthToken,
```

Comment:
If DataPlaneObservabilityConfig is not enabled then `AccessToken` will be set to an empty string? Is that what we want? If so, :+1:

Reply:
Yes, an empty string is fine here. That's also consistent with the other secrets.